[BUG] Error creating bean with name 'userBlizzardAccountController'

- org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'userBlizzardAccountController': Unsatisfied dependency expressed through field 'service'; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'userBlizzardAccountServiceImpl': Unsatisfied dependency expressed through field 'repository'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userBlizzardAccountRepository' defined in com.amplifier.repositories.UserBlizzardAccountRepository defined in @EnableJpaRepositories declared on JPAConfiguration: Invocation of init method failed; nested exception is org.springframework.data.repository.query.QueryCreationException: Could not create query for public abstract com.amplifier.models.UserBlizzardAccount com.amplifier.repositories.UserBlizzardAccountRepository.findById(int)! Reason: Validation failed for query for method public abstract com.amplifier.models.UserBlizzardAccount com.amplifier.repositories.UserBlizzardAccountRepository.findById(int)!; nested exception is java.lang.IllegalArgumentException: Validation failed for query for method public abstract com.amplifier.models.UserBlizzardAccount com.amplifier.repositories.UserBlizzardAccountRepository.findById(int)!

- Caused by: org.hibernate.hql.internal.ast.QuerySyntaxException: user_blizzard_accounts is not mapped [SELECT u from user_blizzard_accounts u where u.id = ?1]
        at org.hibernate.hql.internal.ast.QuerySyntaxException.generateQueryException(QuerySyntaxException.java:79) ~[hibernate-core-jakarta-5.6.9.Final.jar:5.6.9.Final]
        at org.hibernate.QueryException.wrapWithQueryString(QueryException.java:103) ~[hibernate-core-jakarta-5.6.9.Final.jar:5.6.9.Final]
        at org.hibernate.hql.internal.ast.QueryTranslatorImpl.doCompile(QueryTranslatorImpl.java:220) ~[hibernate-core-jakarta-5.6.9.Final.jar:5.6.9.Final]
        at org.hibernate.hql.internal.ast.QueryTranslatorImpl.compile(QueryTranslatorImpl.java:144) ~[hibernate-core-jakarta-5.6.9.Final.jar:5.6.9.Final]
        at org.hibernate.engine.query.spi.HQLQueryPlan.<init>(HQLQueryPlan.java:113) ~[hibernate-core-jakarta-5.6.9.Final.jar:5.6.9.Final]
        at org.hibernate.engine.query.spi.HQLQueryPlan.<init>(HQLQueryPlan.java:73) ~[hibernate-core-jakarta-5.6.9.Final.jar:5.6.9.Final]
        at org.hibernate.engine.query.spi.QueryPlanCache.getHQLQueryPlan(QueryPlanCache.java:162) ~[hibernate-core-jakarta-5.6.9.Final.jar:5.6.9.Final]
        at org.hibernate.internal.AbstractSharedSessionContract.getQueryPlan(AbstractSharedSessionContract.java:636) ~[hibernate-core-jakarta-5.6.9.Final.jar:5.6.9.Final]
        at org.hibernate.internal.AbstractSharedSessionContract.createQuery(AbstractSharedSessionContract.java:748) ~[hibernate-core-jakarta-5.6.9.Final.jar:5.6.9.Final]
        ... 85 common frames omitted

--- a/src/main/java/com/amplifier/repositories/UserBlizzardAccountRepository.java
+++ b/src/main/java/com/amplifier/repositories/UserBlizzardAccountRepository.java
@@ -14,17 +14,17 @@ import org.springframework.transaction.annotation.Transactional;
 @Transactional
 public interface UserBlizzardAccountRepository extends JpaRepository<UserBlizzardAccount, Integer> {
 
-    @Query("SELECT * FROM user_blizzard_accounts")
+    @Query(value = "SELECT * FROM user_blizzard_accounts", nativeQuery = true)
     public List<UserBlizzardAccount> findAll();
 
-    @Query("SELECT u from user_blizzard_accounts u where u.id = ?1")
+    @Query(value = "SELECT u from user_blizzard_accounts u WHERE u.id = ?1", nativeQuery = true)
     public UserBlizzardAccount findById(int id);
 
     @Modifying
-    @Query("UPDATE user_blizzard_accounts u set u.account_name = ?1 where id = ?2")
+    @Query(value = "UPDATE user_blizzard_accounts u SET u.account_name = ?1 where id = ?2", nativeQuery = true)
     public boolean update(String accountName, int id);
 
-    @Query("DELETE * FROM user_blizzard_accounts where id=?1")
+    @Query(value = "DELETE * FROM user_blizzard_accounts WHERE id=?1", nativeQuery = true)
     public boolean delete(int id);
 
 }

